sensor_msgs/RegionOfInterest roi
int32 id
